app/pages/lab-pages-editor/components/TasksPage/components/EditStepDialog/types/DrawingTask.jsx

```diff
@@ -319,14 +327,28 @@ function DrawingTask({
         </span>
       </div>
       <div className="input-row">
-        <label
-          className="big spacing-bottom-S"
-          htmlFor={`task-${taskKey}-help`}
-        >
-          Help Text
-        </label>
+        <div className="flex-row spacing-bottom-S">
+          <label
+            className="big"
+            htmlFor={`task-${taskKey}-help`}
+          >
+            Help Text
+          </label>
+          <button
+            aria-label={`Show/Hide Help field`}
+            className="plain"
+            onClick={toggleShowHelpField}
+            type="button"
+          >
```

Comment:
```suggestion
          <button
            aria-label={`Show/Hide Help field`}
            aria-controls={`task-${taskKey}-help`}
            aria-expanded={showHelpField ? 'true' : 'false'}
            className="plain"
            onClick={toggleShowHelpField}
            type="button"
          >
```

See the [WAI-ARIA disclosure widget pattern](https://www.w3.org/WAI/ARIA/apg/patterns/disclosure/#wai-ariaroles,states,andproperties) for the appropriate states for a disclosure toggle button. The state is currently conveyed visually, by the icon, but not programmatically in the DOM.

Reply:
I _think_ `aria-controls` pointing to the text input is correct here. I've never used it to point directly to a form input before. Usually, it would point to an wrapper element eg. from a tab to the tabpanel that it controls.